src/server/api/medias/bruincast.js

```diff
@@ -32,6 +32,17 @@ router.post('/crosslists', (req, res) => {
 });
 
 router.get('/casts', (req, res) => {
+  const roles = res.locals.token.roles.map(role =>
+    role.substr(role.lastIndexOf('#') + 1, role.length).toLowerCase()
+  );
+  if (
+    !roles.includes('learner') &&
+    !roles.includes('teacher') &&
+    !roles.includes('instructor') &&
+    !roles.includes('administrator')
+  ) {
+    return res.status(400);
```

Comment:
@maxpluez Error 400 is Bad Request. It makes more sense to return 403, which is Forbidden